<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>analysisPage</class>
 <widget class="QWidget" name="analysisPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="sendReportButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Отправить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>380</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Возврат</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="reportBrowser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>421</width>
     <height>271</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;q = d / v&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;r = sum (rk*n)&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;h = r / sum (r)&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;d = sum (x*h)&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="strtglabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Стратегия:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="strategyLine">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>300</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="calcButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>220</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="crisisLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Значение критерия x:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="x_criteriaValueLine">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>130</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="crisisLabel_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>300</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Оценка стратегии q:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="q_strategyValueLine">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>300</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="targetLabel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Оценка цели d:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="d_targetValueLine">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>170</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="crisisLabel_4">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>130</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Общий вес r:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rk_commonWeightLine">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>130</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="criteriaLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вес критерия rk:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rk_criteriaWeightLine">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>80</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="markLabel">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>80</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Кол-во оценок n:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="n_marksNumber">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>80</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="checkButton">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>220</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Проверить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Цель:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="targetNameValue">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>170</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
